birdvoxpaint/display.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+
+def specshow(S, sr, period=60, offset=0,
+             segment_duration=10, date_offset=None,
+             fmin=0, fmax=None, y_axis='mel', sp_height=None):
+    '''Plot a false color spectrogram (or any long spectrogram for that matter).
+
+    Arguments:
+        S (np.ndarray):
+        sr (int): 
+    '''
+    nfreq, nseg, nidxs = S.shape
+    if nidxs == 1: # use cmap
+        S = S[:,:,0]
```

Comment:
So this shows a single acoustic index? Whichever is indexed as `0`?

Reply:
I would have used `S.squeeze()`, but I didn't want it to fall over if someone naively entered an `S` with shape `(256, 1, 1)`

Reply:
ah yes, my bad, sorry i misread.